diff --git "a/C:\\Users\\david\\AppData\\Local\\Temp\\TortoiseGit\\Mat9675.tmp\\Math-a4cf325-left.h" "b/C:\\work\\code\\oslbuildwin\\phase4\\llvm\\include\\llvm\\CodeGen\\PBQP\\Math.h"
index 69a9d83..7d2ed9d 100644
--- "a/C:\\Users\\david\\AppData\\Local\\Temp\\TortoiseGit\\Mat9675.tmp\\Math-a4cf325-left.h"
+++ "b/C:\\work\\code\\oslbuildwin\\phase4\\llvm\\include\\llvm\\CodeGen\\PBQP\\Math.h"
@@ -138,7 +138,7 @@ private:
 
 class VectorComparator {
 public:
-  bool operator()(const Vector &A, const Vector &B) {
+  bool operator()(const Vector &A, const Vector &B) const {
     if (A.Length < B.Length)
       return true;
     if (B.Length < A.Length)
@@ -386,7 +386,7 @@ private:
 
 class MatrixComparator {
 public:
-  bool operator()(const Matrix &A, const Matrix &B) {
+  bool operator()(const Matrix &A, const Matrix &B) const {
     if (A.Rows < B.Rows)
       return true;
     if (B.Rows < A.Rows)
