diff --git "a/C:\\Users\\david\\AppData\\Local\\Temp\\TortoiseGit\\idiff-e50010f.001.cpp" "b/C:\\work\\code\\oslbuildwin\\phase3\\oiio\\src\\idiff\\idiff.cpp"
index 8ced491..b2c624f 100644
--- "a/C:\\Users\\david\\AppData\\Local\\Temp\\TortoiseGit\\idiff-e50010f.001.cpp"
+++ "b/C:\\work\\code\\oslbuildwin\\phase3\\oiio\\src\\idiff\\idiff.cpp"
@@ -307,7 +307,7 @@ main (int argc, char *argv[])
 // when Visual Studio is used float values in scientific foramt are 
 // printed with three digit exponent. We change this behaviour to fit
 // Linux way
-#ifdef _MSC_VER
+#if _MSC_VER < 1900
                 _set_output_format(_TWO_DIGIT_EXPONENT);
 #endif
                 std::streamsize precis = std::cout.precision();
