diff --git "a/C:\\Users\\david\\AppData\\Local\\Temp\\TortoiseGit\\oiiotool-e50010f.000.cpp" "b/C:\\work\\code\\oslbuildwin\\phase3\\oiio\\src\\oiiotool\\oiiotool.cpp"
index 02e4f6e..40a4621 100644
--- "a/C:\\Users\\david\\AppData\\Local\\Temp\\TortoiseGit\\oiiotool-e50010f.000.cpp"
+++ "b/C:\\work\\code\\oslbuildwin\\phase3\\oiio\\src\\oiiotool\\oiiotool.cpp"
@@ -4369,7 +4369,7 @@ main (int argc, char *argv[])
 {
 // When Visual Studio is used float values in scientific format are printed 
 // with three digit exponent. We change this behavior to fit the Linux way.
-#ifdef _MSC_VER
+#if _MSC_VER < 1900
     _set_output_format (_TWO_DIGIT_EXPONENT);
 #endif
 
